Ginger/Ginger/Actions/ActionEditPage.xaml.cs

```diff
@@ -2259,5 +2259,34 @@ protected override void IsVisibleChangedHandler(object sender, DependencyPropert
                 base.IsVisibleChangedHandler(sender, e);
             }
         }
+
+        private void xTimeoutTextBox_PreviewTextInput(object sender, TextCompositionEventArgs e)
+        {
+
+            bool isEmptyString = string.IsNullOrEmpty(e.Text);
+            if (isEmptyString)
+            {
+                //allow
+                e.Handled = false;
+                return;
+            }
+
+            string currentText = xTimeoutTextBox.Text != null ? xTimeoutTextBox.Text : string.Empty;
```

Comment:
allow setting timeout to empty

Reply:
Yes it's allowing-
Basically this code is checking-
Allows input if the input text is empty.
Allows input if the resulting text (current text + new input) forms a valid integer.
Disallows any input that does not result in a valid integer.